shell/src/main/java/alluxio/shell/command/CopyFromLocalCommand.java

```diff
@@ -87,6 +87,9 @@ public void run(CommandLine cl) throws AlluxioException, IOException {
    */
   private void copyFromLocalDir(File srcDir, AlluxioURI dstPath)
       throws AlluxioException, IOException {
+    if (!srcDir.canRead()) {
+      throw new IOException(srcDir + " (Permission denied)");
```

Comment:
Is this guaranteed to be caused by permission denied, or could there be other causes, e.g. the file not existing? Maybe we could use `IOException("Cannot read " + srcDir)` instead